packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/ios/summaryofsteps.md

```diff
@@ -0,0 +1,36 @@
+## Working with the sample code
```

Comment:
Same comments as before in previous article reviews.

Reply:
Updated. See previous comments.

Reply:
Yup yup. Resolve.